client/signup/steps/import-url/index.jsx

```diff
@@ -55,34 +64,55 @@ class ImportURLStepComponent extends Component {
 		this.props.goToNextStep();
 	};
 
-	handleInputChange = value => {
+	handleInputChange = event => {
+		const value = get( event, 'target.value', '' );
+
 		this.props.setNuxUrlInputValue( value );
-		this.debouncedFetchIsSiteImportable( normalizeUrlForImportSource( value ) );
+		this.checkValidation( value );
 	};
 
-	fetchIsSiteImportable = () => {
-		const normalizedUrl = normalizeUrlForImportSource( this.props.urlInputValue );
-		this.props.fetchIsSiteImportable( normalizedUrl );
+	checkValidation( value ) {
+		const message = this.getValidationMessage( value );
+
+		// If there is a validation message, show it after
+		// a delay so as to give the user chance to finish typing.
+		if ( message ) {
+			this.debouncedSetValidationMessage( message );
+		} else {
+			// If the input is good, feedback immediately
+			this.debouncedSetValidationMessage.cancel();
+			this.setValidationMessage( '' );
+		}
+	}
+
+	setValidationMessage = message => {
+		this.props.setValidationMessage( message );
 	};
 
-	debouncedFetchIsSiteImportable = debounce( this.fetchIsSiteImportable, DEBOUNCE_CHECK_INTERVAL );
+	debouncedSetValidationMessage = debounce( this.setValidationMessage, VALIDATION_INTERVAL );
 
 	renderContent = () => {
-		const { isInputDisabled, urlInputValue, translate } = this.props;
+		const { isInputDisabled, urlInputValidationMessage, urlInputValue, translate } = this.props;
+		const canContinue = urlInputValue && ! urlInputValidationMessage;
 
 		return (
 			<div className="import-url__wrapper">
-				<FormTextInputWithAction
+				<FormTextInput
 					placeholder="Enter the URL of your existing site"
 					action="Continue"
-					onAction={ this.handleAction }
 					label={ translate( 'URL' ) }
 					onChange={ this.handleInputChange }
 					disabled={ isInputDisabled }
 					value={ urlInputValue }
+					isError={ urlInputValidationMessage }
```

Comment:
I'm getting PropTypes validation error. `isError` is supposed to be a boolean

Reply:
Ahh of course, I'll cast it! Weird that I didn't see that message 🤔